addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -952,9 +952,6 @@ registerWebsitePreviewTour("website_form_editable_content", {
         trigger: ":iframe .o_overlay_move_options .o_move_handle",
         run: "drag_and_drop :iframe section.s_website_form",
     },
-    {
-        trigger: ":iframe section.s_website_form .col-lg-4[contenteditable=true]",
-    },
```

Comment:
I think it is a mistake, if I remember it well we put that recently to fix a race condition. It definitely needed a comment though, strange we merged it that way. Something like
```
// TODO we should ensure that as soon as the snippet is visible, it is editable to avoid this step
```
In any case, just keep this step please :pray: